example/lib/queued_interceptor_crsftoken.dart

```diff
@@ -1,92 +1,95 @@
-import 'dart:async';
-import 'dart:convert';
+import 'dart:developer';
 
 import 'package:dio/dio.dart';
 
+/// Used for Web Cookie
+// const String _cookieKey = 'XSRF_TOKEN';
+const String _headerKey = 'X-Csrf-Token';
+
 void main() async {
-  final dio = Dio();
-  //  dio instance to request token
-  final tokenDio = Dio();
-  String? csrfToken;
-  dio.options.baseUrl = 'https://seunghwanlytest.mocklab.io/';
-  tokenDio.options = dio.options;
+  String? cachedCSRFToken;
+
+  final dio = Dio()
+    ..options.baseUrl = 'https://httpbin.org/'
```

Comment:
Use `httpbun.com` instead.

Reply:
OK, changed baseUrl and used [mix](https://httpbun.com/#mix) 👍